### PR TITLE
Monkey powder

### PR DESCRIPTION
**Pull request by Hopekz**
Ports Monkey powder from TG.
Unsure which PR as I cannibalized it from their codebase directly.

What it does:
Monkey cubes can now be ground down to a powder. 
This powder, when combined with water, will also make monkies.

#### Changelog

:cl:  Hopek
rscadd: Added Monkey powder. Obtainable from grinding monkey cubes.
tweak: Monkey cubes now grind into Monkey powder rather than Nutriment.
tweak: Gorilla cubes now grind into Monkey powder rather than Nutriment.
/:cl:

### CODE REVIEW

**Comment by overlord12345** [comment]
Time to snort ape dust

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -543,6 +543,23 @@
...
+	name = /datum/reagent/monkey_powder
+	id = /datum/reagent/monkey_powder
+	results = list(/datum/reagent/monkey_powder = 3)
+	required_reagents = list(/datum/reagent/consumable/banana = 1, /datum/reagent/consumable/nutriment=2,/datum/reagent/liquidgibs = 1)
+
+/datum/chemical_reaction/monkey
+	name = "monkey"
+	id = "monkey"
+	required_reagents = list(/datum/reagent/monkey_powder = 30, /datum/reagent/water = 1)
+
```
Comment (ynot01) [inline code/modules/reagents/chemistry/recipes/others.dm:557]:
```suggestion
    mob_react = FALSE
    
```
if you dont want players shitting out monkeys

Comment (Hopekz) [reply code/modules/reagents/chemistry/recipes/others.dm:557]: I kind of like players being able to give birth to monkeys via a reaction in their body.
Willing to accept input though.

Comment (boodaliboo) [reply code/modules/reagents/chemistry/recipes/others.dm:557]: same

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 2]
```diff
@@ -543,6 +543,23 @@
...
+/datum/chemical_reaction/monkey/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	new /mob/living/carbon/monkey(location)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	if (location) new /mob/living/carbon/monkey(location)
```

**Review by Ktlwjec1 — APPROVED** [review]

**Pull request merged**